src/platform/datapath_raw_xdp.c

```diff
@@ -47,6 +48,7 @@ typedef struct XDP_QUEUE {
     XSK_RING TxRing;
     XSK_RING TxCompletionRing;
     BOOL Error;
+    XDP_WORKER* XdpWorker;
```

Comment:
Let's put this at the top, after `Interface` and just call this `Worker`. Can it be `const` too?